_layouts/docwithnav.html

```diff
@@ -12,21 +12,38 @@
 	{% include documentation.html %}
 </div>
 <script type="text/javascript">
-	$(document).ready(function() {
-		/* Find and set target="_blank" for all external URLs (in navigation menu and content body) so that they open in a new tab/window */
-        $('a').filter(function() {
-			return this.hostname && this.hostname !== location.hostname;
-		}).attr('target', '_blank');
-	});
+    $(document).ready(function() {
+		/* On page load, open the navigation menu and highlight the menu item that corresponds to the current URL */
+		var curnavitem = $('#wrapper a[href="' + window.location.href + '"]');
```

Comment:
Change this from window.location.pathname to window.location.href (to find absolute URLs instead of just the suffix - result of difference in local build config)